pages/linux/bzip2.md

```diff
@@ -0,0 +1,15 @@
+# bzip2
+
+> A block-sorting file compressor.
+
+- Compress file:
+
+`bzip2 {{path/to/file_to_compress}}`
+
+- Decompress file:
+
+`bzip2 -d {{path/to/compressed_file.bz2}}`
+
+- Decompress to STDOUT:
+
+`bzip2 -dc {{path/to/compressed_file.bz2}}`
```

Comment:
Actually from the manual:
`       -c --stdout`
            Compress or decompress to standard output.

`      -d --decompress`
Force decompression.  bzip2, bunzip2 and bzcat are really the same program, and the decision about what actions to take is done on the basis of which name is used.  This flag overrides that mechanism, and forces bzip2 to decompress.


-c parameter is quite ambiguous, when I try to compress data to STDOUT I get 
`bzip2: I won't write compressed data to a terminal.`